.travis.yml

```diff
@@ -6,3 +6,18 @@ services:
 
```

Comment:
Can you please give few words in the commit message about what was the issue about?

Reply:
There was no curl.

Reply:
Added this info to the cmt msg as well